src/partials/layout.html

```diff
@@ -43,53 +43,54 @@
   </head>
   <body data-page="{{ page }}">
     <!-- Navbar desktop -->
-    <header
-      class="m-auto w-full max-w-7xl px-4 py-6 hidden lg:flex items-center"
-    >
-      <div class="w-56">
-        <a href="/">
-          <img
-            src="/images/logo_with_text.svg"
-            alt=""
-            height="45"
-            width="188"
-          />
-        </a>
-      </div>
-      <div class="flex-grow flex items-center justify-center">
-        <nav class="flex space-x-12 text-gray-600">
-          <a href="/#install">Install</a>
-          <a href="/#features">Features</a>
-          <a href="/integrations.html">Integrations</a>
-          <a class="announcekit-widget" href="https://news.livebook.dev/"
-            >What’s new?&nbsp</a
-          >
-        </nav>
-      </div>
-      <div class="w-56 flex justify-end">
-        <nav class="flex space-x-3 items-center">
-          <a
-            class="px-3 py-2 rounded-lg font-semibold text-sm bg-blue-600 border border-transparent text-white hover:bg-blue-700 focus:bg-blue-700 flex items-end space-x-1.5"
-            href="/badge.html"
-          >
-            <img src="/images/badge_logo.svg" alt="" width="17" height="19" />
-            <span class="mb-[-1.5px] font-semibold">Create badge</span>
-          </a>
-          <a
-            class="p-2 flex items-center justify-center rounded-lg bg-white border border-gray-300 text-gray-500 hover:bg-gray-100 focus:bg-gray-100 text-xl leading-none"
-            href="/settings.html"
-            aria-label="settings"
-          >
-            <i class="ri-settings-3-line"></i>
+    <header class="header-fixed">
```

Comment:
```suggestion
    <header id="header" class="header-fixed">
```

so we can consistently query for `#header` and `#header-mobile` from JS.